.circleci/config.yml

```diff
@@ -0,0 +1,48 @@
+version: 2.1
+orbs:
+  go: gotest/tools@0.0.13
+
+executors:
+  golang:
+    docker:
+      - image: circleci/golang:1.16.3
+    resource_class: 2xlarge
+
+commands:
+  checkout-dealbot:
+    steps:
+      - checkout
+  install-deps:
+    steps:
+      - go/install-ssh
+      - go/install: {package: git}
+  install-lotus:
+    steps:
+      - run: sudo apt-get update
+      - run: sudo apt-get install ocl-icd-opencl-dev libhwloc-dev
+      - run: git clone https://github.com/filecoin-project/lotus.git /tmp/lotus
+      - run: cd /tmp/lotus && git submodule sync && git submodule update --init
+      - run: cd /tmp/lotus && make debug && sudo make install
+      - run: cd /tmp/lotus && sudo cp lotus-seed /usr/local/bin
+  test-dealbot:
+    steps:
+      - run: go build -o ./devnet-binary cmd/devnet/*.go
+      - run: go build -o dealbot-binary
+      - run: sudo cp devnet-binary /usr/local/bin/devnet
+      - run: sudo cp dealbot-binary /usr/local/bin/dealbot
+      - run: ./integration_tests/01_storage_retrieval_ok.sh
```

Comment:
We might need to give a bit more than 30sec. for the devnet to start up when running on CI. At least on the `Docker Medium` VM I am getting on my personal fork, it takes longer - we should review once this is merged and running on `2xlarge`.